### PR TITLE
v1.9.1 .github - VERSION RELEASE main - CHANGELOG.md to RST

### PR DESCRIPTION
**Pull request by mindyls**
# Related Issues
required to close NRLMMD-GEOIPS/geoips#171

# Reviewer Instructions
* Please confirm merge is into 'main' branch from 'v1.9.1-release'
* Review "Files changed" tab
* Confirm appropriate testing was completed for these updates.

# Summary
Changes for version 1.9.1 update on repo .github.
* Update CHANGELOG.md to CHANGELOG.rst in PR and review templates.

See NRLMMD-GEOIPS/geoips#171 for additional repositories included in this update.

# Post Merge Steps
Once this PR has been merged, v1.9.1 will be tagged/released on the .github main branch.

### CODE REVIEW

File: docs/source/releases/CHANGELOG.md [hunk 1]
```diff
@@ -10,10 +10,36 @@
...
+  * Testing Updates
+  * Documentation Updates
+  * Changes During Release Process
+  * Developmental Updates
+  * Incoming Changes from GitHub.com
 
 # v1.6.0: 2022-11-28, open source release
 ## GEOIPS#119: 2022-11-28, open source release
-* Update VERSION to 1.6.0
+* Update VERSION to 1.5.4
```
Comment (cpcamacho) [inline docs/source/releases/CHANGELOG.md:42]: @mindyls I don't think it much matters since the VERSION file is going away, but shouldn't that stay 1.6.0?

**Review by cpcamacho — APPROVED** [review]
This looks good to go after my comment is resolved!

**Review by jsolbrig — APPROVED** [review]
lgtm

File: docs/source/releases/CHANGELOG.md (hunk 1 above)
Comment (jsolbrig) [reply docs/source/releases/CHANGELOG.md:42]: I don't think we should worry about this at this point. It's old enough that it doesn't matter much IMO.

Comment (mindyls) [reply docs/source/releases/CHANGELOG.md:42]: Ignoring!

**Pull request merged**